src/stdlib/PairUtils.scillib

```diff
@@ -34,3 +35,21 @@ let snd : forall 'A. forall 'B. Pair 'A 'B -> 'B =
     match p with
     | Pair a b => b
     end
+
+let pair_eq : forall 'A. forall 'B. ('A -> 'A -> Bool) -> ('B -> 'B -> Bool) ->
```

Comment:
Unfortunately, we cannot just extend the existing standard library modules with new names (here is a related issue  https://github.com/Zilliqa/scilla/issues/818). I suggest to do pattern-matching in `CryptoUtils.scillib` instead.